troposphere/static/js/components/modals/instance/launch/components/ResourcesForm.jsx

```diff
@@ -135,13 +135,11 @@ const ResourcesForm = React.createClass({
         </label>
         );
     },
-    render: function() {
-        let { identity, identityList, onIdentityChange, providerSize, providerSizeList, onSizeChange, } = this.props;
-        return (
-        <form>
-            {globals.USE_ALLOCATION_SOURCES
-             ? this.renderAllocationSourceMenu()
-             : null}
+    renderIdentity() {
```

Comment:
I think it makes the situation for `render` more intentionally clear. 

I offer a dissenting opinion even though I _yield_:

It seems to cut against the encapsulations of the method, from my view. It I was going to use it from multiple invocation locations - then I'd have to sprinkle in this "conditional". **But**, I favor this suggestion because the method is really intended to be "private" (or privileged) usage from within the component.